src/components/WalletModal/index.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { ApplicationModal } from 'state/application/reducer'
+
+import { render, screen } from '../../test-utils'
+import WalletModal from './index'
+
+beforeEach(() => {
+  delete global.window.ethereum
+})
+
+afterAll(() => {
+  delete global.window.ethereum
+})
+
+const UserAgentMock = jest.requireMock('utils/userAgent')
+jest.mock('utils/userAgent', () => ({
+  isMobile: false,
+}))
+
+jest.mock('.../../state/application/hooks', () => {
+  return {
+    useModalOpen: (_modal: ApplicationModal) => true,
+    useWalletModalToggle: () => {
+      return
+    },
+  }
+})
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    useWeb3React: () => ({
+      account: undefined,
+      isActive: false,
+      isActivating: false,
+      connector: undefined,
+    }),
+    ...web3React,
+  }
+})
+
+test('Loads Wallet Modal on desktop', async () => {
+  render(<WalletModal pendingTransactions={[]} confirmedTransactions={[]} />)
+  expect(screen.getByText('Install MetaMask')).toBeInTheDocument()
+  expect(screen.getByText('Coinbase Wallet')).toBeInTheDocument()
+  expect(screen.getByText('WalletConnect')).toBeInTheDocument()
+  expect(screen.getByText('Fortmatic')).toBeInTheDocument()
+  expect(screen.getAllByTestId('wallet-modal-option')).toHaveLength(4)
+})
+
+test('Loads Wallet Modal on desktop with MetaMask installed', async () => {
+  global.window.ethereum = { isMetaMask: true }
+
+  render(<WalletModal pendingTransactions={[]} confirmedTransactions={[]} />)
+  expect(screen.getByText('MetaMask')).toBeInTheDocument()
+  expect(screen.getByText('Coinbase Wallet')).toBeInTheDocument()
+  expect(screen.getByText('WalletConnect')).toBeInTheDocument()
+  expect(screen.getByText('Fortmatic')).toBeInTheDocument()
+  expect(screen.getAllByTestId('wallet-modal-option')).toHaveLength(4)
+})
+
+test('Loads Wallet Modal on mobile', async () => {
+  UserAgentMock.isMobile = true
+
+  render(<WalletModal pendingTransactions={[]} confirmedTransactions={[]} />)
+  expect(screen.getByText('Open in Coinbase Wallet')).toBeInTheDocument()
+  expect(screen.getByText('WalletConnect')).toBeInTheDocument()
+  expect(screen.getByText('Fortmatic')).toBeInTheDocument()
+  expect(screen.getAllByTestId('wallet-modal-option')).toHaveLength(3)
+})
+
+test('Loads Wallet Modal on MetaMask browser', async () => {
```

Comment:
Can we change these test names to things like "it('loads wallet modal on MetaMask browser'....". The 'it' is an alias for test and its supposed to be a naming convention like "it ... is supposed to behave like this". 
All the other tests in the repo use 'it' currently so just want to keep it consistent.